packages/slate/src/interfaces/editor.ts

```diff
@@ -1265,6 +1265,13 @@ export const Editor = {
       at: end,
       match: n => Editor.isBlock(editor, n),
     })
+
+    // If last element is a void node, unhang range by including void node in range
+    if (Editor.isVoid(editor, endBlock[0])) {
```

Comment:
I am not sure this the best solution.  Should we just simply not unhang the range in `Transforms.delete` if the selection end is inside a void node?  I think maybe we should handle this in `Transforms.delete` vs. changing the default behavior of `unhangRange`?  Let's pair on it.

Reply:
I thought there were some path issues before, but I didn't realize that they're handled further down in `Transforms.delete`. I've moved the end void node handling to `Transforms.delete` so that it now skips `unhangRange`. If you think there are still changes that could be made I'm happy to pair. Thanks for the feedback!